README.md

```diff
@@ -32,11 +32,12 @@ ldcli --help
 
 The LaunchDarkly CLI allows you to save preferred settings, either within a config file using the `config` commands, or set as environment variables.
 
-Current respected settings:
+Current settings:
```

Comment:
Updated the readme to be consistent with the config help output.

Reply:
How about:

```suggestion
Supported settings:
```
?

Reply:
Updated